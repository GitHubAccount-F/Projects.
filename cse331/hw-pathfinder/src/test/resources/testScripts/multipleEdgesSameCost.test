#Tests Dijkstra's algorithm given edges with the same
#cost

#Creates graph
CreateGraph graph1

#Adds nodes
AddNode graph1 one
AddNode graph1 two

#Adds Edges
AddEdge graph1 one two 7.5
AddEdge graph1 one two 7.5

#Finds path
FindPath graph1 one two