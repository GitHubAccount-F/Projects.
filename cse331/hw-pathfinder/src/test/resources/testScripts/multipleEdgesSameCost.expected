#Tests Dijkstra's algorithm given edges with the same
#cost

#Creates graph
created graph graph1

#Adds nodes
added node one to graph1
added node two to graph1

#Adds Edges
added edge 7.500 from one to two in graph1
added edge 7.500 from one to two in graph1

#Finds path
path from one to two:
one to two with weight 7.500
total cost: 7.500